_wiki/FAQ/LinuxAppSpecific.md

```diff
@@ -25,3 +25,29 @@ env SDL_VIDEODRIVER=wayland ./osu.AppImage
 ```bash
 env SDL_VIDEODRIVER=wayland osu-lazer
 ```
+
+---
+
+### Art Programs {#art}
+
+Firstly, make sure your output mode is [set to Artist Mode]({% link _wiki/FAQ/Linux.md %}#artist-mode %})
+
+You can confirm the setting is applied by running the following command and verifying that at least 1 listed device contains 'Artist' in the name.
+```bash
+libinput list-devices | grep -i OpenTabletDriver
+```
+
+If your program still does not work after ensuring the output mode has been set in the driver, please read on.
+
+#### GIMP {#gimp}
+
+1. Go to 'Edit -> Input Devices' in the GIMP Toolbar
+2. Make sure any of the devices matching the following settings are set to 'Mode: Screen' in the right hand panel
+    - `Core Pointer`
+    - `xwayland-tablet eraser`, if present
+    - `xwayland-tablet stylus`, if present
+    - `OpenTabletDriver Virtual Artist Tablet` (FIXME: verify this is correct for X11)
```

Comment:
This is how it appears on X11 GNOME 44
![image](https://github.com/OpenTabletDriver/opentabletdriver.github.io/assets/39218853/de37897e-d2cb-4662-a724-b936f0bdd9da)

Reply:
Thanks, updating PR with this.